metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/GetProfile.java

```diff
@@ -132,36 +157,26 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(GetProfile.class);
 
-  /**
-   * A client that can retrieve profile values.
-   */
+  // Cached client that can retrieve profile values.
```

Comment:
Um, you got me.  Perhaps my IDE "helped" me with this one.  Since the member variable being documented is private, there was no real reason to switch to a Javadoc-style comment.  The point of the edit was to add the word "cached" to the comment.  If I make any other tweaks, I'll change it back to a "//" comment.

There is a rule, which is that Javadoc comments should be used with public interfaces, and it doesn't matter which you use with private interfaces, altho no harm in using the Javadocs format if you want the documentation of a private i/f to be really clear.